sdk/tests/tutorials/valuation/test_differing_quote_txns_ids.py

```diff
@@ -0,0 +1,237 @@
+import unittest
+import json
+from datetime import datetime
+import pytz
+import uuid
+
+import lusid
+import lusid.models as models
+from utilities import PortfolioLoader
+from utilities import TestDataUtilities
+
+
+class IdentifiersRecipe(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Create a configured API client
+        api_client = TestDataUtilities.api_client()
+
+        # Setup required LUSID APIs
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        cls.aggregation_api = lusid.AggregationApi(api_client)
+        cls.quotes_api = lusid.QuotesApi(api_client)
+        cls.recipes_api = lusid.ConfigurationRecipeApi(api_client)
+
+        # Setup test parameters
+        cls.effective_date = datetime(2019, 4, 15, tzinfo=pytz.utc)
+
+        # Setup test data from utilities
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+        # Setup test portfolio
+        cls.portfolio_scope = TestDataUtilities.tutorials_scope
+        cls.portfolio_code = cls.test_data_utilities.create_transaction_portfolio(
+            TestDataUtilities.tutorials_scope
+        )
+
+        # Load transactions to test portfolio
+        portfolio_loader = PortfolioLoader(
```

Comment:
would it make sense to use the id generator, see https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/tests/tutorials/ibor/test_portfolios.py#L41 for an example?